docs/markdown/Contributions/development/style-guide.md

```diff
@@ -17,31 +17,29 @@ $ build-support/githooks/pre-commit
 ```
```

Comment:
I don't see any changes beyond white space? This is kind of disruptive. (Ik it's IDE-generated)

Reply:
Lines 25 and 26